src/types.ts

```diff
@@ -1,4 +1,4 @@
-export type UnitSelectType = 'default' | 'grid' | 'flex'
+export type UnitSelectType = 'default' | 'grid' | 'flex' | 'size' | 'auto'
```

Comment:
What is 'auto' here?

Reply:
my bad! I added to test something and I forget to delete it